DataManagementSystem/scripts/dirac-dms-replicate-and-register-request.py

```diff
@@ -106,13 +107,15 @@ def getLFNList( arg ):
       gLogger.error( "unable to put request '%s': %s" % ( request.RequestName, putRequest["Message"] ) )
       error = -1
       continue
-
+    requestIDs.append( str( putRequest["Value"] ) )
     if not multiRequests:
       gLogger.always( "Request '%s' has been put to ReqDB for execution." % request.RequestName )
 
   if multiRequests:
     gLogger.always( "%d requests have been put to ReqDB for execution, with name %s_<num>" % ( count, requestName ) )
-  gLogger.always( "You can monitor requests' status using command: 'dirac-rms-show-request <requestName>'" )
+  if len(requestIDs) > 0:
```

Comment:
Hi,

Thanks for the PR !
The prefered way of such tests according to PEP8 is ```if requestIDs:```

After this fix, review OK